src/graphics/device.js

```diff
@@ -359,6 +363,30 @@ Object.assign(pc, function () {
             gl.FLOAT
         ];
 
+        this.pcUniformType = {};
```

Comment:
I would prefer internal properties to start with _

Reply:
I'm fine with that, but it'll have to wait for another PR since almost no member variables in device.js have underscore. A full refactor to address this and create consistency would take some time.